packages/Python/lldbsuite/test/lang/swift/generic_error_tuple/main.swift

```diff
@@ -0,0 +1,29 @@
+class MyErr : Error {
+  var x : Int
+
+  init(_ x : Int) {
+    self.x = x
+  }
+}
+
+class MyOtherErr : MyErr {}
+
+func h<U, V>(_ tuple : (U, V)) -> (U, V) {
+  return tuple //%self.expect("frame var -d run-target -- tuple", 
```

Comment:
I just noticed that, for some reason, RemoteAST thinks the dynamic type of this is the base class `MyErr` and not the derived class `MyOtherErr`. It's not a bug in lldb, I think, but I'm going to open a rdar to investigate.